src/components/fields/Array/index.tsx

```diff
@@ -27,6 +28,7 @@ export const config: IFieldConfig = {
     popoverProps: { PaperProps: { sx: { p: 1, minWidth: "200px" } } },
   }),
   SideDrawerField,
+  filter: { operators },
   requireConfiguration: false,
```

Comment:
```suggestion
  SideDrawerField,
  filter: { operators, defaultValue: [] },
  requireConfiguration: false,
```

This is to improve user experience.